UnityProject/Assets/Scripts/Clothing/Eyewear/SecurityHUD.cs

```diff
@@ -0,0 +1,316 @@
+			{
+				NewJobType = idCard.JobType;
+				break;
+			}
+			else if (itemSlot.Item.TryGetComponent<Items.PDA.PDALogic>(out var pda))
+			{
+				idCard = pda.GetIDCard();
+				if (idCard == null) continue;
+				NewJobType = idCard.JobType;
+				break;
+			}
+		}
+
+		var newJobIcon = JobTypeToIcon(NewJobType);
+
+		//Look up in database
+		SyncCurrentJob(CurrentJob, newJobIcon);
+	}
+
+
+	public StatusIcon SecurityStatusToIcon(SecurityStatus securityStatus)
+	{
+		switch (securityStatus)
+		{
+			case SecurityStatus.None:
+				return StatusIcon.None;
+			case SecurityStatus.Arrest:
+				return StatusIcon.Wanted;
+			case SecurityStatus.Criminal:
+				return StatusIcon.Incarcerated;
+			case SecurityStatus.Parole:
+				return StatusIcon.Released;
+			default:
+				return StatusIcon.Paroled;
+		}
+
+	}
+
+	public JobIcon JobTypeToIcon(JobType JobType)
+	{
+		switch (JobType)
+		{
+			case JobType.NULL:
+				return JobIcon.NoID;
+			case JobType.ASSISTANT:
+				return JobIcon.Assistant;
+			case JobType.ATMOSTECH:
+				return JobIcon.AtmosphericTechnician;
+			case JobType.BARTENDER:
+				return JobIcon.Bartender;
+			case JobType.BOTANIST:
+				return JobIcon.Botanist;
+			case JobType.CAPTAIN:
+				return JobIcon.Captain;
+			case JobType.CARGOTECH:
+				return JobIcon.CargoTech;
+			case JobType.CENTCOMM_INTERN:
+			case JobType.CENTCOMM_OFFICER:
+			case JobType.CENTCOMM_COMMANDER:
+				return JobIcon.Centcom;
+			case JobType.CHAPLAIN:
+				return JobIcon.Chaplin;
+			case JobType.CHEMIST:
+				return JobIcon.Chemist;
+			case JobType.CHIEF_ENGINEER:
+				return JobIcon.ChiefEngineer;
+			case JobType.CMO:
+				return JobIcon.ChiefMedicalOfficer;
+			case JobType.CLOWN:
+				return JobIcon.Clown;
+			case JobType.COOK:
+				return JobIcon.Cook;
+			case JobType.CURATOR:
+				return JobIcon.Curator;
+			case JobType.DETECTIVE:
+				return JobIcon.Detective;
+			case JobType.GENETICIST:
+				return JobIcon.Geneticist;
+			case JobType.HOP:
+				return JobIcon.HeadOfPersonnel;
+			case JobType.HOS:
+				return JobIcon.HeadOfSecurity;
+			case JobType.JANITOR:
+				return JobIcon.Janitor;
+			case JobType.LAWYER:
+				return JobIcon.Lawyer;
+			case JobType.DOCTOR:
+				return JobIcon.Medic;
+			case JobType.MIME:
+				return JobIcon.Mime;
+			case JobType.PRISONER:
+				return JobIcon.Prisoner;
+			case JobType.QUARTERMASTER:
+				return JobIcon.QuarterMaster;
+			case JobType.RD:
+				return JobIcon.ResearchDirector;
+			case JobType.ROBOTICIST:
+				return JobIcon.Roboticist;
+			case JobType.SCIENTIST:
+				return JobIcon.Scientist;
+			case JobType.SECURITY_OFFICER:
+				return JobIcon.Security;
+			case JobType.MINER:
+				return JobIcon.ShaftMiner;
+
+			case JobType.ENGINEER:
+				return JobIcon.StationEngineer;
+
+			case JobType.VIROLOGIST:
+				return JobIcon.Virologist;
+
+			case JobType.WARDEN:
+				return JobIcon.Warden;
+			default:
+				return JobIcon.Unknown;
+		}
+	}
+
+
+	public void Awake()
```

Comment:
put unity functions at the top of functions

Reply:
sounds good though do we have anywhere where we have example layouts?

Reply:
I'll update the code guidelines to point that out.